package.json

```diff
@@ -22,25 +22,30 @@
     "author": "Adheesh Bhatia",
     "license": "BSD-3-Clause",
     "dependencies": {
-        "@dhis2/d2-ui-header-bar": "^1.1.1",
+        "@dhis2/app-runtime": "^2.0.4",
+        "@dhis2/ui-widgets": "^5.0.1",
+        "babel-polyfill": "^6.26.0",
         "d2": "^30.0.0",
         "d2-ui": "^29.0.29",
         "material-ui": "^0.20.0",
         "prop-types": "^15.5.10",
-        "react": "^16.3.0",
-        "react-dom": "^16.2.0",
+        "react": "^16.13.0",
+        "react-dom": "^16.13.0",
         "react-router": "^3.2.1",
-        "rxjs": "^5.5.7"
+        "rxjs": "^5.5.7",
+        "styled-jsx": "^3.3.0"
```

Comment:
I think you can remove this. It's not used directly in the app anywhere it seems.

Reply:
the app breaks without this